source/guides/references/experiments.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Experiments
+---
+
+If you'd like to try out what we're working on, you can enable beta features for your project by setting configuration using the options below.
+
+{% note warning %}
+⚠️ The experimental features might change or ultimately be removed without making it into the core product. Our primary goal for experiments is to collect real-world feedback during the development.
+{% endnote %}
+
+When you open a Cypress project, clicking on the **Settings** tab and clicking into the **Experiments** panel will display the experimental features that are available and whether they are enabled for your project.
+
+Option | Default | Description
+----- | ---- | ----
+`experimentalGetCookiesSameSite` | `false` | If `true`, Cypress will add `sameSite` values to the objects yielded from {% url "`cy.setCookie()`" setcookie %}, {% url "`cy.getCookie()`" getcookie %}, and {% url "`cy.getCookies()`" getcookies %}. This will become the default behavior in Cypress 5.0.
+`experimentalComponentTesting` | `false` | When set to `true`, Cypress allows you to execute component tests using framework-specific adaptors. By default  `cypress/component` is the path for component tests. You can change this setting by setting the `componentFolder` configuration option. For more details see the {% url "cypress-react-unit-test" https://github.com/bahmutov/cypress-react-unit-test/tree/feature/cypress-mount-mode %} and {% url "cypress-vue-unit-test" https://github.com/bahmutov/cypress-vue-unit-test/tree/feature/cypress-mount-mode %} repos.
+
+An example component test would look like this
+
+```js
+import { mount } from 'cypress-react-unit-test'
+import Post from './Post.jsx'
+describe('Post skeletons', () => {
+  it('loads title after timeout', () => {
+    mount(<Post title={title} children={text} />)
+    // and then use regular Cypress commands
+    // at first, the title and the text are 💀
+    cy.get('h1 .react-loading-skeleton').should('have.length', 1)
+    cy.get('p .react-loading-skeleton').should('have.length', 5)
+    ...
+  })
+})
+```
+
+This test would execute in the browser, similar to the full end-to-end test, except there is no website URL:
+
+{% imgTag /img/guides/references/component-test.gif "Example React component test" %}
+
+Feature | Test libraries | Cypress component tests
+--- | --- | ---
+Test runs in real browser | ❌ | ✅
+Uses full mount | ❌ | ✅
```

Comment:
React Testing Library does a full mount. https://kentcdodds.com/blog/why-i-never-use-shallow-rendering

>  I've expressed this feeling on many occasions and get asked on a regular basis why I feel the way I do about shallow rendering and why React Testing Library will never support shallow rendering.

Reply:
Enzyme is the only one who doesn't do a full mount.. and that's optional.

Reply:
regardless of the mount - i don't think this is a real differentiator.

it's not so much about full or partial mount, or "whether it runs in a real browser"...

it's really about **seeing your components** and natively interacting with them. we should call to attention that you can see your components just like if you were in dev with a browser open pointing to `localhost`.

we should also call to attention videos and screenshots "just work" out of the box. no need to deal with an artificial DOM. it's the same DOM and the same interaction and rendering engine. there's no translation layer